src/resource.coffee

```diff
@@ -114,12 +116,19 @@ class Resource extends Model
         ids ?= if Array.isArray resourceLink
           resourceLink
 
+        href = null if type is 'users'
```

Comment:
My bad I was debugging something. I'll remove that.

On June 17, 2015 4:55:26 PM CDT, Brian Carstensen <notifications@github.com> wrote:
>> @@ -114,12 +116,19 @@ class Resource extends Model
>>          ids ?= if Array.isArray resourceLink
>>            resourceLink
>>  
>> +        href = null if type is 'users'
>
>Why're users special here? I'd like to keep any knowledge about the
>data itself out of the library.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/brian-c/json-api-client/pull/2/files#r32679516

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.